packages/core/src/common/utils.ts

```diff
@@ -50,6 +50,33 @@ export function clamp(val: number, min: number, max: number) {
     return Math.min(Math.max(val, min), max);
 }
 
+/* Alternate implmentation of Element.closest */
+export function closest(element: Node, selector: string) {
```

Comment:
Why isn't the native one good enough? If this is for polyfill/compatibility goodness, is there any way to fall back to the native?